OneSignalSDK/onesignal/core/src/test/java/com/onesignal/core/internal/operations/OperationRepoTests.kt

```diff
@@ -485,6 +490,54 @@ class OperationRepoTests : FunSpec({
             executor.execute(withArg { it[0] shouldBe secondOp })
         }
     }
+
```

Comment:
Not in the change but the test at line 459 failed in these commits. It passed in main branch.

Reply:
I don't' understand on two points:
1. This test? "ensure results from executeOperations are added to beginning of the queue"?
2. Assuming you mean the test from 1 above, see the test passes here in this first commit: https://github.com/OneSignal/OneSignal-Android-SDK/actions/runs/8728583277/job/23948694395#step:6:779

Reply:
I'll address this in a different PR, since this flaky test wasn't added in this PR.